[BUG] panic: runtime error: slice bounds out of range [1:0]

when `infoSchemaColInfo.ColumnDefault` is `'`, The code will throw the following exception: 

```bash
panic: runtime error: slice bounds out of range [1:0]

goroutine 1 [running]:
github.com/smallnest/gen/dbmeta.cleanupDefault({0x102c15f98, 0x1})
        /Users/<username>/.mountroot/www/go/gen/dbmeta/meta_utils.go:225 +0x3dc
github.com/smallnest/gen/dbmeta.LoadMysqlMeta(0x140003acc30, {0x16dbf3764, 0x5}, {0x16dbf37d2, 0x10}, {0x14000428d1b, 0x5})
        /Users/<username>/.mountroot/www/go/gen/dbmeta/meta_mysql.go:75 +0x6ac
github.com/smallnest/gen/dbmeta.LoadMeta({0x16dbf3764, 0x5}, 0x140003acc30, {0x16dbf37d2, 0x10}, {0x14000428d1b, 0x5})
        /Users/<username>/.mountroot/www/go/gen/dbmeta/meta.go:315 +0xa8
github.com/smallnest/gen/dbmeta.LoadTableInfo(0x140003acc30, {0x1400044f400, 0x38, 0x40}, {0x0, 0x0, 0x0}, 0x14000370800)
        /Users/<username>/.mountroot/www/go/gen/dbmeta/meta.go:694 +0x1ac
main.main()
        /Users/<username>/.mountroot/www/go/gen/main.go:272 +0xff8

Process finished with the exit code 2

```

--- a/dbmeta/meta_utils.go
+++ b/dbmeta/meta_utils.go
@@ -207,6 +207,10 @@ where table_schema = '%s' AND
 }
 
 func cleanupDefault(val string) string {
+	if len(val) < 2 {
+		return val
+	}
+
 	if strings.HasPrefix(val, "(") && strings.HasSuffix(val, ")") {
 		return cleanupDefault(val[1 : len(val)-1])
 	}

